cmd/infra/aws/iam.go

```diff
@@ -211,7 +211,6 @@ const (
         "ec2:ModifyInstanceAttribute",
         "ec2:ModifyNetworkInterfaceAttribute",
         "ec2:ModifySubnetAttribute",
-        "ec2:ReleaseAddress",
```

Comment:
can you please also remove AllocateAddress
and remove them from the API docs https://github.com/openshift/hypershift/blob/main/api/v1beta1/hostedcluster_types.go#L1285?